.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -155,9 +160,41 @@ jobs:
         name: sycl_lit_${{ inputs.build_artifact_suffix }}
         path: lit.tar.xz
 
-  llvm_test_suite:
+  aws-start-matrix:
+    name: Start AWS Matrix
     needs: build
     if: ${{ inputs.lts_matrix != '' }}
+    strategy:
+      fail-fast: false
+      max-parallel: ${{ inputs.max_parallel }}
+      matrix:
+        include: ${{ fromJSON(inputs.lts_matrix) }}
+    runs-on: ubuntu-latest
+    environment: aws
+    steps:
+      - uses: actions/checkout@v3
+        if: ${{ matrix.aws-type }}
+        with:
+          path: llvm
+      - run: npm install ./llvm/devops/actions/aws-ec2
+        if: ${{ matrix.aws-type }}
+      - name: Start AWS EC2 runner
+        if: ${{ matrix.aws-type }}
+        uses: ./llvm/devops/actions/aws-ec2
+        with:
+          label: ${{ matrix.runs-on }}
+          GH_PERSONAL_ACCESS_TOKEN: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
+          AWS_ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY }}
+          AWS_SECRET_KEY: ${{ secrets.AWS_SECRET_KEY }}
+          aws-ami:  ${{ matrix.aws-ami }}
+          aws-spot: ${{ matrix.aws-spot }}
+          aws-type: ${{ matrix.aws-type }}
+          aws-disk: ${{ matrix.aws-disk }}
+          one-job:  ${{ matrix.one-job }}
+
+  llvm_test_suite:
```

Comment:
It looks like currently 6 llvm_test_suite tasks wait for all 6 aws-start tasks to complete, same is true for aws-stop tasks.
Ideally we should try to make the whole sequence aws-start -> llvm-test-suite run -> aws-stop independent from each other.
It would be more cost effective in case aws-start task waits for completion for too long.

Reply:
Yes, we will see how common there ubuntu-latest instance spawn delays are without max-parallel first before changing that into single job.

Reply:
Changed to one job here.